new-frontiers-14/frontier-station-14
# Cry-O Feedback Tweaks

## About the PR



- Removes redundant carbon and oxygen from Salicylic Acid
- Adjusts Aloxadone caustic heal from 1 to 3
- Adds additional Aloxadone recipe replacing Sig with Siderlac.

## Why / Balance


Ongoing feedback regarding cryo medicine and the bounty system has routinely settled on the challenges of sustaining interest in attempting to recover bodies with certain types of damage due to the return not justifying the resources. The returns seem to be in a good place to me and the feedback has been consistent enough to warrant investigation.

- Salicylic Acid is a homebrew precursor that appears to be weighted down by redundant carbon and oxygen needs. Oxygen and carbon are factored into two other components of the recipe, in one place as a core ingredient, and do not impact the end yield. Removing them streamlines the recipe and better aligns the end result with expectations for input. If this is considered undesirable I would suggest adjusting the yield to 5 over 3.

- Aloxadone caustic restoration has been increased from 1 to 3. This was a major focus point for addressing damages of this type as the drain on resources seemed unnecessarily high. This brings the healing in line with the other damage types addressed. Which seems appropriate given that the primary medicines involved directly address caustic damage.

- Aloxadone receives a secondary recipe. Switching SIG for Siderlac. Siderlac is a botanical compound derived from Galaxy Thistle and Aloe, both of which are found in the medicine seed crate. Siderlac heals caustic damage, as does SIG. I believe that this will allow for chemists to create initial batches of Aloxadone from materials on hand and then swap over to trickling in sustaining quantities of meds as the shift progresses if more direct methods are being bottlenecked by resources.

## How to test


Check the recipes. Check that they're reflected in the guidebook. Give them a test.

## Media


## Requirements



## Breaking changes


Nada

**Changelog**


🆑 
- tweak: Slight adjustments to Salicylic Acid recipe, additional recipe for Aloxadone and 2 point buff to Aloxadone caustic damage heal.
